catbuffer/schemas/nem/mosaic/mosaic_definition.cats

```diff
@@ -76,6 +76,7 @@ struct MosaicDefinition
 	properties = array(SizePrefixedMosaicProperty, properties_count)
 
 	# size of the serialized levy
+	@sizeref(levy, 4)
```

Comment:
hmm, actually (asking cause I don't remember), is this +4 or +0?
(what will generator do if value of this field - when deserializing - is 0?)

Reply:
+4